rails-tasks/app/views/people/tasks.html.erb

```diff
@@ -0,0 +1,32 @@
+<div class="on_paper">
+<h1><%= @person.name %></h1>
+to complete:
+<ul>
+  <% @complete_tasks_array = [] %>
+  <% @person.tasks.each do |task| %>
+      <% if task.complete == true &&   @completed_date == nil%>
+        <% @complete_tasks_array.push(task.name) %>
+      <% elsif task.complete == true %>
+        <% @complete_tasks_array.push(task.name) %>
```

Comment:
I think this `elsif` case is making the `@completed_date == nil` check on line 7 redundant. It looks like we're doing the same thing if `task.complete == true` regardless.

I like the idea of building the list of completed tasks in the same loop where you're outputting the incomplete tasks!